doc/Articles/FAQ.md

```diff
@@ -133,6 +133,21 @@ NetworkServer.connections.Count
 - Socket connections, includes people without spawned prefabs, non authenticated, or that may have bugged out but during spawn, but have connected temporarily (Android users minimising game).  Only host / server can check this.
 
 NetworkManager.singleton.numPlayers
-- Number of active spawned player objects across all connections on the server.
-Only host / server can check this, below is an example of how to sync this number to all players.
+- Number of active spawned player objects across all connections on the server (only the host / server can check this)
+</details>
+
+### Changing Transports
+
+<details>
+   <summary>Changing transports is very easy and only requires a few steps</summary>
 
+- Go to the GameObject that has a NetworkManager attached
+- Remove the current transport script
+- Add another transport script via the Add Component button
+- Drag the transport script to the "Transport" field in the NetworkManager
```

Comment:
```suggestion
- Add another transport script via the Add Component button
- Drag the new transport script to the "Transport" field in the `NetworkManager`
- Remove the current transport script (optional)
```
Re-ordered steps because Network Manager may not allow removal of the only transport, or will auto-add it again.